backend/src/models/User.interface.ts

```diff
@@ -1,4 +1,4 @@
-export interface IRegisterUserRq{
+export interface IUserRq{
```

Comment:
separate interface of login and user register

Reply:
Done , Sir!

Reply:
Change to ILoginRq

Reply:
Change to ILoginRq